### ISSUE TITLE
[BUG] Add & update File Property in pages

### ISSUE THREAD

**Issue by laHeud** [mariosimao/notion-sdk-php#219]
**Describe the bug**
Hi, I have an error message when I tried to created / add a file in an existing properties.

```
        $file = File::createExternal($propertyValue);
        $property = \Notion\Pages\Properties\Files::create($file);
        $updatedPage = $page->addProperty($propertyName, $property);


        return $this->client->pages()->update($updatedPage);
```

`body failed validation. Fix one:
body.properties.Gitlab.files[0].file should be defined, instead was `undefined`.
body.properties.Gitlab.files[0].name should be defined, instead was `undefined`.`

I can update this properties with Postman. I have compared the body between Postman and my code and it seems that only one this change the "id" = "" became name="Gitlab". It works when I use the name with Postman but didn't work otherwise.

An idea ?

**Comment by mariosimao** [comment]
Hi @laHeud! It looks like a poorly documented behaviour of the Notion API.

The [File object documentation](https://developers.notion.com/reference/file-object) does not indicate at all the existence of a `name` property. However, the [File property documentation](https://developers.notion.com/reference/page-property-values#files) briefly shows the usage of `name` on the examples.

I will be adding a `name` property to the `File` common class.

Example of usage when the development is complete:

```php
$file = File::createExternal("https://bitcoin.org/bitcoin.pdf")->changeName("My file name");
$property = Files::create($file);

$page = $page->addProperty("File", $property);

$notion->pages()->update($page);

```

**Comment by mariosimao** [comment]
@laHeud I have just released [version 1.8.0](https://github.com/mariosimao/notion-sdk-php/releases/tag/v1.8.0) which fixes this bug.
